lib/Common/DataStructures/SparseBitVector.h

```diff
@@ -535,7 +535,7 @@ BVSparse<TAllocator>::GetNextBit(BVSparseNode *node) const
             return ret + node->startIndex;
         }
     }
-    return -1;
+    return (BVIndex)-1;
```

Comment:
>(BVIndex)-1 [](start = 11, length = 11)

BVInvalidIndex? (and in lines 533, 552, and 567)

Reply:
Addressed, as well as in two locations in preproc macros at the top of the file.